CMakeLists.txt

```diff
@@ -6,6 +6,11 @@ option(LSP_BUILD_ASAN "Build with ASAN" OFF)
 option(LSP_STATIC_CRT "Link with the static CRT (/MT)" OFF)
 option(LSP_BUILD_WITH_SENTRY "Build with Sentry (crash reporting)" OFF)
 
+set(LSP_VERSION "1.49.1")
+set(LSP_NAME "Luau Language Server")
```

Comment:
Feel free to change this to `luau-lsp` or whatever name your prefer. It can be a constant inside the `src/LanguageServer.cpp`, but I thought that keeping these two together makes sense